src/SDKs/TrafficManager/Management.TrafficManager/Microsoft.Azure.Management.TrafficManager.csproj

```diff
@@ -1,25 +1,25 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-	<!-- Please do not move/edit code below this line -->
-	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
-	<!-- Please do not move/edit code below this line -->
-
-	<PropertyGroup>
-		<PackageId>Microsoft.Azure.Management.TrafficManager</PackageId>
-		<Description>Microsoft Azure Management TrafficManager Library</Description>
-		<AssemblyName>Microsoft.Azure.Management.TrafficManager</AssemblyName>
-		<Version>2.4.0-preview</Version>
-		<PackageTags>Microsoft Azure TrafficManager management;TrafficManager;TrafficManager management;</PackageTags>
-        <PackageReleaseNotes>
-            <![CDATA[
-                This is a preview release of the Azure Traffic Manager SDK.
-            ]]>
-        </PackageReleaseNotes>
-	</PropertyGroup>
-	<PropertyGroup>
-		<TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
-	</PropertyGroup>
-
-	<!-- Please do not move/edit code below this line -->
-	<Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.RP.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.RP.props'))" />
-	<!-- Please do not move/edit code above this line -->
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.Management.TrafficManager</PackageId>
+		<Description>Microsoft Azure Management TrafficManager Library</Description>
+		<AssemblyName>Microsoft.Azure.Management.TrafficManager</AssemblyName>
+		<Version>2.5.0</Version>

```

Comment:
@allencal Please confirm whether you are ready to release a stable version

Reply:
I am ready to release a stable version.